src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -725,9 +728,17 @@ private static bool TryDeserializeJson<TResult>(string json, out TResult result)
                 }
                 return Task.FromResult<object>(null);
             });
+
+            var console = (Func<object, Task<object>>)(msg =>
+            {
+                _consoleLogger?.LogDebug((string)msg);
```

Comment:
They are not, but I've had a lot of feedback in the past to reduce chattiness and discourage use of console.log, hence opt in. I'd prefer on by default.